internal/tools/proto/go.mod

```
@@ -0,0 +1,9 @@
+module github.com/DataDog/datadog-agent/internal/tools/proto
+
+go 1.16
+
+require (
+	github.com/golang/mock v1.5.0 // indirect
+	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.12.2 // indirect
```

I think none of these should have the `// indirect` comment since they are explicitly imported on the `tools.go` file

```suggestion
	github.com/golang/mock v1.5.0
	github.com/golang/protobuf v1.3.2
	github.com/grpc-ecosystem/grpc-gateway v1.12.2
```

---

Oh, I added those with `go get` to set the pins, otherwise it defaulted to other versions. But yeah, we don't need the `indirect` directives.